sllurp/llrp.py

```diff
@@ -171,6 +191,7 @@ def __init__(self, factory, duration=None, report_every_n_tags=None,
                  mode_identifier=None,
                  session=2, tag_population=4,
                  tag_filter_mask=None,
+                 config_dict=DEFAULT_READER_CONFIG,
```

Comment:
How about moving it even one step further out, to `LLRPClientFactory`?

Reply:
Oh, great idea!

Reply:
Done in e4a6da0.